src/include/stat.h

```diff
@@ -274,12 +274,13 @@ __wt_stats_clear(void *stats_arg, int slot)
  * milliseconds or microseconds. Also values less than a given minimum are ignored and not put in
```

Comment:
This comment is no longer correct - would you be able to remove the last two sentences from this paragraph, please?

(I'm not too concerned about the mentioned problem of excessively large smallest values - if it becomes a problem, we can revert the change.)

Reply:
hi, @korteland 
I have delete the no longer correct comment.

I have calculated that it will not exceed the limit, assuming that each request takes 1000us and 100,000 requests per second, this parameter can be used for 5,849 years(18446744073709551615/1000/10000/3600/24/365=5849years).

thanks

Reply:
Thanks @y123456yz ! This looks good to me now.